packages/design-system/src/components/web-components/ds-autocomplete/index.ts

```diff
@@ -0,0 +1 @@
+export * from './ds-autocomplete';
```

Comment:
Testing out the astro example, I was expecting the `dist/web-components/bundle/ds-autocomplete.js` import to work as well—not just the main bundle. I found that if I changed this line to the following, that individual component bundle works too:

```suggestion
import './ds-autocomplete';
```

Reply:
Great, thanks for looking into the _why_. I had been using the main bundle when testing locally.